plugins/ccgx/fu-ccgx-dmc-device.c

```diff
@@ -616,33 +621,26 @@ fu_ccgx_dmc_device_attach(FuDevice *device, FuProgress *progress, GError **error
 		return TRUE;
 
 	if (self->update_model == DMC_UPDATE_MODEL_DOWNLOAD_TRIGGER) {
-		DmcTriggerCode trigger_code = DMC_TRIGGER_CODE_UPDATE_NOW;
-
-		if (manual_replug)
-			trigger_code = DMC_TRIGGER_CODE_UPDATE_ON_DISCONNECT;
-
-		if (!fu_ccgx_dmc_device_send_download_trigger(self, trigger_code, error)) {
-			g_prefix_error(error, "download trigger error: ");
-			return FALSE;
-		}
+		if (self->trig_code > 0) {
+			if (!fu_ccgx_dmc_device_send_download_trigger(self, self->trig_code, error)) {
+				g_prefix_error(error, "download trigger error: ");
+				return FALSE;
+			}
+			if (self->trig_code == 2) /* trigger code for update on disconnect */
```

Comment:
Isn't this just DMC_TRIGGER_CODE_UPDATE_ON_DISCONNECT -- which was removed in this commit?

Reply:
> Isn't this just DMC_TRIGGER_CODE_UPDATE_ON_DISCONNECT -- which was removed in this commit?
 
I will remove this code and add has-manual-replug to the quirk file for the device that support update on disconnect